src/runtime/virtcontainers/qemu_amd64.go

```diff
@@ -173,6 +185,10 @@ func (q *qemuAmd64) cpuModel() string {
 		cpuModel += ",vmx=off"
 	}
 
+	if q.sgxEPCSize != 0 {
```

Comment:
Just from reading nearby code -- is there any limitation with respect to migration and SGX?

Reply:
https://www.qemu.org/docs/master/system/i386/sgx.html#migration

Reply:
thanks @mythi , live migration is supported but guest software stack should support recreating enclaves when it suffers sudden lose of EPC

Reply:
Got it. Either way, if an operator / user is configuring SGX, trying to use any of they templating logic, they will have their work cut out for them!